src/dlgProfilePreferences.cpp

```diff
@@ -189,6 +189,14 @@ dlgProfilePreferences::dlgProfilePreferences(QWidget* pF, Host* pHost)
                                                "<li><b>Partly checked</b> <i>(Default) 'auto'</i> = Use the setting that the system provides.</li></ul></p>"
                                                "<p><i>This setting is only processed when individual menus are created and changes may not "
                                                "propogate everywhere until Mudlet is restarted.</i></p>"));
+    // Set the tooltip on the containing widget so both the label and the
+    // control have the same tool-tip:
+    widget_timerDebugOutputMinimumInterval->setToolTip(QStringLiteral("<html><head/><body>%1</body></html>")
+                                                         .arg(tr("<p>If a timer has only a short interval it can quickly fill up the <i>Central Debug Console</i> "
+                                                                 "windows with messages that it ran okay on each occasion it is called; this (per profile) "
+                                                                 "control adjusts a threshold that will mute the debug output for those timers which "
+                                                                 "executed <b>correctly</b> but have an interval less than the duration set; this can be used "
+                                                                 "so that just other more useful debug output to <i>that</i> display can be seen.</p>")));
```

Comment:
:+1: though I would word it more concisely - people want to know what they need to know, fast -

> Set the minimum interval for timer information to show up in the debug console

Reply:
I don't want people to misunderstand and think it will stop the timers from producing any results elsewhere in the profile - it ONLY shuts up the messages about the timer code running *correctly* in the Debug console (if the timer produces an error each time it runs then there WILL be a lot of repeated **error** message...! :scream:) Also, if the timer produces some text output each time it runs anyway then that is going to spam the normal console or wherever it sends it's output to - but that is the scripter's problem to handle, not this control!

Reply:
Kebap has a good suggestion on this that you missed: https://github.com/Mudlet/Mudlet/pull/1903#discussion_r211086979